app/views/layouts/application.html.erb

```diff
@@ -29,8 +29,16 @@
   <body class="flex flex-col min-h-screen h-full bg-gray-100">
     <div class="flex-1 bg-gray-100">
       <%= render partial: 'layouts/nav' %>
-
-      <div class="py-10">
+      <div class="pt-2">
+        <div class="max-w-7xl mx-auto sm:px-6 lg:px-8">
+          <% if request.path == '/offers' %>
```

Comment:
There has to be a better way of doing this than looking at the path. Perhaps use content_  tags to do it.

Reply:
I will look into it